kateryna/daemon/log_reader_daemon.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+SCRIPT=./parsing_logs.sh
+RUNAS=root
+PIDFILE=./my_command.pid
+LOGFILE=./my_command.log
+
+start() {
+   if [ -f "$PIDFILE" ] && kill -0 $(cat "$PIDFILE"); then
+   echo 'Service already running' >&2
+    return 1
+  fi
+  echo 'Starting service…' >&2
+  echo $$  > "$PIDFILE"
+  local CMD="$SCRIPT &> \"$LOGFILE\" & echo \$!"
+  su -c "$CMD" $RUNAS > "$PIDFILE"
```

Comment:
It doesn't work on Ubuntu too, if try to run as:

`./log_reader_daemon.sh start`
or `sudo ./log_reader_daemon.sh start`.


It just more or less work in case of:
`sudo su`
`./log_reader_daemon.sh`
But it also raises an error:

```
./log_reader_daemon.sh: 8: kill: Usage: kill [-s sigspec | -signum | -sigspec] [pid | job]... or
kill -l [exitstatus]
```

Well, it should be executable via:
`./log_reader_daemon.sh start`
or
`sudo sh log_reader_daemon.sh start`
or
`sudo ./log_reader_daemon.sh start`